### ISSUE TITLE
Accessibility: Volume and Set Playback Rate controls are not operable with keyboard when focusing

### ISSUE THREAD

**Issue by mreyk** [polimediaupv/paella#322]
# Accessibility bug

Paella Player does not fullfil the Guideline 2.1 of the WCAG:

[https://www.w3.org/TR/WCAG21/#keyboard-accessible](https://www.w3.org/TR/WCAG21/#keyboard-accessible)

### Volume control

1. Enable screen reader (Tested with ChromeVox 53.0.2784.5 in Google Chrome and Ubuntu)
2. Play a video
3. Navigate through controls with Tab key
4. Reach the Volume control. The screen reader reads "Volume, button".
5. Press the button (Enter key)
6. The volume control bar is shown, but there is no key able to up or down volume.

### Set Playback Rate control

1. Enable screen reader (Tested with ChromeVox 53.0.2784.5 in Google Chrome and Ubuntu)
2. Play a video
3. Navigate through controls with Tab key
4. Reach the Set Playback Rate control. The screen reader reads "Set Playback Rate, button".
5. Press the button (Enter key)
6. The playback rate choices are shown, but there is no key able to choice the desire playback rate.

## Actual Behavior:

The volume control bar is shown, but there is no key able to up or down volume. 
The playback rate choices are shown, but there is no key able to choice the desire playback rate.
 
## Expected Behavior:

When selecting volume control, allow to up and down volume with keyboard.
When selecting Set Playback Rate control, allow the screen reader to read available playback rate choices and chose the desire one with keyboard.

**Comment by paulgration** [comment]
This issue seems to have been closed prematurely as the fix commit only addresses controlling the volume bar from a keyboard but not the playback rate...

Ideally when selecting the volume bar using a screen reader it should notify the user that they are controlling the volume/what the current level is

Also when pressing enter/return to bring up the volume bar, pressing return again doesn't close the popup - the popup only disappears when pressing enter/return on another popup menu item.

Aside from volume bar and playback rate there are `zoom`, `video quality`, `extended tab adaptor items`, `sharing` and possibly other popup items that should be accessible from the keyboard..these may need new/separate issues?
